### DISCUSSION TITLE
Error when trying to import from a file/module named site

### DISCUSSION THREAD

**Discussion by creatixdevelopers** [fastapi/fastapi#6220] [Questions] [answered]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
----- main.py -----
from fastapi import FastAPI

from site import site_router

app = FastAPI()
app.include_router(site_router)


----- site.py -----
from fastapi import APIRouter

site_router = APIRouter(prefix='/')


@site_router.get('/')
async def site_route():
    return {'msg': 'site'}
```

### Description

When I try to import routes from a module or file named 'site', I get the following error:

`Fatal Python error: init_import_site: Failed to import the site module`
`Python runtime state: initialized`
`Traceback (most recent call last):`
`  File "path-to-my-project\site.py", line 1, in <module>`
`    from fastapi import APIRouter`
`ModuleNotFoundError: No module named 'fastapi'`

### Operating System

Windows

### Operating System Details

Edition	Windows 10 Home Single Language
Version	21H2
OS build	19044.1741
Experience	Windows Feature Experience Pack 120.2212.4180.0

### FastAPI Version

0.78.0

### Python Version

3.10.0

**Comment by odiseo0** [comment]
Do you have other versions of Python installed? I don't think this is a FastAPI error though

**Comment by jeer2234** [comment]
both files are at the same path?

**Comment by creatixdevelopers** [comment, marked as answer]
Turns out  [this](https://docs.python.org/3/library/site.html) inbuilt python module was interfering with my code. I managed to resolve the issue by moving my project into a module of it's own.
